main.js

```diff
@@ -77,20 +77,20 @@ const log = report => {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 };
 
+const files = (report, predicate) => report.results
+	.filter(predicate)
```

Comment:
Yea you are right, it focuses the last opened file. Didn't thought about it...

@sindresorhus Should the warnings only be opened if there are no errors?